third_party/blink/renderer/core/css/style_engine.h

```diff
@@ -886,11 +886,14 @@ class CORE_EXPORT StyleEngine final : public GarbageCollected<StyleEngine>,
 
   Member<CSSFontSelector> font_selector_;
 
-  HeapHashMap<AtomicString, WeakMember<StyleSheetContents>>
-      text_to_sheet_cache_;
+  HeapHashMap<AtomicString, WeakMember<StyleSheetContents>> text_to_sheet_cache_;
   HeapHashMap<WeakMember<StyleSheetContents>, AtomicString>
       sheet_to_text_cache_;
 
+  // Fix weak pointer divergence.
+  // see https://linear.app/replay/issue/RUN-1065#comment-7e345cbc
+  HeapHashSet<Member<StyleSheetContents>> record_replay_contents_strong_;
```

Comment:
@bhackett1024 
1) I was following your example in `animation_timeline.h` for naming. But yeah, I can change it.
2) I'm not sure we need to add all too many other explanations, since it *does* fix weak pointer divergence, and, for details, check the link?

Reply:
-> `style_sheet_contents_strong_`